brave-snouts-frontend/src/app/app.module.ts

```diff
@@ -69,6 +74,8 @@ import { HotToastModule } from '@ngneat/hot-toast';
     
     // Shared
     MaterialModule,
+    ClipboardModule,
```

Comment:
Check if this is duplicate and already located in material module

Since this is angular material module. I want it in material module